.builds/freebsd.yml

```diff
@@ -16,22 +16,24 @@ sources:
 - https://github.com/neovim/neovim
 
 tasks:
-- build-deps: |
-    cd neovim
-    gmake deps
-- build-debug: |
-    mkdir -p neovim/build/debug
-    cd neovim/build/debug
-    cmake -G Ninja -DTRAVIS_CI_BUILD=ON -DCMAKE_BUILD_TYPE=Debug -DMIN_LOG_LEVEL=3 ../..
-    ninja -v nvim unittest-prereqs functionaltest-prereqs
-- build-release: |
-    mkdir -p neovim/build/release
-    cd neovim/build/release
-    cmake -G Ninja -DTRAVIS_CI_BUILD=ON -DCMAKE_BUILD_TYPE=Release -DMIN_LOG_LEVEL=3 ../..
-    ninja -v nvim unittest-prereqs functionaltest-prereqs
-- test-debug: |
-    cd neovim/build/debug
-    ninja -v unittest functionaltest
-- test-release: |
-    cd neovim/build/release
-    ninja -v unittest functionaltest
+- get-environment: |
+    env
+# - build-deps: |
+#     cd neovim
+#     gmake deps
+# - build-debug: |
+#     mkdir -p neovim/build/debug
+#     cd neovim/build/debug
+#     cmake -G Ninja -DTRAVIS_CI_BUILD=ON -DCMAKE_BUILD_TYPE=Debug -DMIN_LOG_LEVEL=3 ../..
+#     ninja -v nvim unittest-prereqs functionaltest-prereqs
+# - build-release: |
+#     mkdir -p neovim/build/release
+#     cd neovim/build/release
+#     cmake -G Ninja -DTRAVIS_CI_BUILD=ON -DCMAKE_BUILD_TYPE=Release -DMIN_LOG_LEVEL=3 ../..
+#     ninja -v nvim unittest-prereqs functionaltest-prereqs
+# - test-debug: |
+#     cd neovim/build/debug
+#     ninja -v unittest functionaltest
+# - test-release: |
+#     cd neovim/build/release
+#     ninja -v unittest functionaltest
```

Comment:
@jszakmeister please use `[ci skip]` in the commit message's subject to skip Travis/AppVeyor builds.

Reply:
@blueyed Ah, it sounded like it could be at the end of the commit message. :-(